app/javascript/components/Ramp.jsx

```diff
@@ -1,46 +1,75 @@
 import React from 'react';
-import { Transcript, IIIFPlayer, MediaPlayer, StructuredNavigation } from "@samvera/ramp";
+import { 
+  Transcript, 
+  IIIFPlayer, 
+  MediaPlayer, 
+  StructuredNavigation, 
+  MetadataDisplay, 
+  SupplmentalFiles 
+} from "@samvera/ramp";
 import 'video.js/dist/video-js.css';
 import "@samvera/ramp/dist/ramp.css";
-import { Col, Row } from 'react-bootstrap';
+import { Col, Row, Tab, Tabs } from 'react-bootstrap';
 import './Ramp.scss';
 
-const Ramp = ({ base_url, mo_id, canvas_count, share, timeline }) => {
-  const [transcriptsProp, setTrancsriptProp] = React.useState([]);
+const Ramp = ({ 
+  base_url, 
+  mo_id, 
+  master_files_count, 
+  share, 
+  timeline, 
+  in_progress, 
+  cdl 
+}) => {
   const [manifestUrl, setManifestUrl] = React.useState('');
+  const { enabled, can_stream, embed, destroy_CDL } = cdl;
```

Comment:
Is there a way to rename the variables locally?   I think it would be helpful for some of these to be prefixed by `cdl` to help provide better context.  Specifically `enabled` and `embed`.  Or maybe we could skip the mapping to access them through the `cdl` object like `cdl.enabled`?  What do you think?

Reply:
Yes, they lose context when doing the mapping from the props. I don't think there's an advantage of using `cdl_` prefix over accessing them through the object. 
Let's go with the format `cdl.enabled` :+1: I'll change the code accordingly.